<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>437</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>533</width>
    <height>437</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="GeneralSettings" name="generalSettings">
      <attribute name="title">
       <string>General</string>
      </attribute>
     </widget>
     <widget class="CompilerSettings" name="compilerSettings">
      <attribute name="title">
       <string>Compiler</string>
      </attribute>
     </widget>
     <widget class="VisualSettings" name="visualSettings">
      <attribute name="title">
       <string>Visual</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GeneralSettings</class>
   <extends>QWidget</extends>
   <header>generalsettings.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CompilerSettings</class>
   <extends>QWidget</extends>
   <header>compilersettings.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VisualSettings</class>
   <extends>QWidget</extends>
   <header>visualsettings.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
